src/jmcomic/jm_downloader.py

```
@@ -1,6 +1,31 @@
 from .jm_option import *
```

Ruff (0.8.2)

`from .jm_option import *` used; unable to detect undefined names

(F403)